engine/src/fungible_token.rs

```diff
@@ -504,8 +504,10 @@ impl<I: IO + Copy> FungibleTokenOps<I> {
 
             self.internal_register_account(account_id);
             let refund = amount - min_balance;
-            if refund > 0 {
-                let action = PromiseAction::Transfer { amount: refund };
+            if refund > ZERO_YOCTO {
+                let action = PromiseAction::Transfer {
+                    amount: refund.as_u128(),
+                };
```

Comment:
Looks like `PromiseAction` enum should be updated to take `Yocto`

Reply:
I've thought about it, but I guess it's not idiomatic. Promises operate on the raw types, since they are direct suppliers to `near-sdk`.

Reply:
I think it should take `Yocto` because it is a typed wrapper over the raw functionality. At the interface where `PromiseAction` is actually converted into a NEAR promise is were we would call `as_u128`. Keeping strong typing throughout all our internal logic can prevent bugs.